### PR TITLE
feat: cache accounts context results

### PR DESCRIPTION
**Pull request by yjkimjunior**
store the following in localstorage

* stashes
* derived balances
* derived staking info

### CODE REVIEW

**Comment by yjkimjunior** [comment]
erm, not sure what happened to AddressSummary component with the new ui-components but it's  gotten....stretchy in a not so nice way. 

not something to åddress in this PR, but, note to self.

**Comment by yjkimjunior** [comment]
also the caching removes some lag on initial render, but the information only loads as fast as AccountsContext is initilaized, which is only as fast as Api is initilaized. still, better than nothing in my experience.

File: front/context/src/AccountsContext.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { Option } from '@polkadot/types';
 import { AccountId, StakingLedger } from '@polkadot/types/interfaces';
 import { logger } from '@polkadot/util';
 import { decodeAddress, encodeAddress } from '@polkadot/util-crypto';
+import { writeStorage } from '@substrate/local-storage';
```
Comment (amaurymartiny) [inline front/context/src/AccountsContext.tsx:17]: Oh, wasn't aware of this. Where is the source code?

File: front/context/src/AccountsContext.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { Option } from '@polkadot/types';
 import { AccountId, StakingLedger } from '@polkadot/types/interfaces';
```
Comment (amaurymartiny) [inline front/context/src/AccountsContext.tsx:14]: fyi I'm using my own [AccountsContext](https://github.com/paritytech/substrate-light-ui/blob/master/packages/light-apps/src/components/ContextGate/context/AccountContext/AccountContext.tsx) in slui, so feel free to modify this one with breaking changes.

File: front/context/src/AccountsContext.tsx (hunk 1 above)
Comment (yjkimjunior) [reply front/context/src/AccountsContext.tsx:17]: https://github.com/yjkimjunior/local-storage

I just forked the rehooks one and made some small edits to a lingering PR to enable it for SSR

File: front/context/src/AccountsContext.tsx (hunk 2 above)
Comment (yjkimjunior) [reply front/context/src/AccountsContext.tsx:14]: ah ok, noted!

File: front/context/src/AccountsContext.tsx (hunk 1 above)
Comment (amaurymartiny) [reply front/context/src/AccountsContext.tsx:17]: Just a thought, not tested: if it's only to make `gatsby build` pass, maybe something like https://www.npmjs.com/package/localstorage-polyfill could be enough

**Pull request merged**